### DISCUSSION TITLE
add icon under image in Person

### DISCUSSION THREAD

**Discussion by Billiardist94** [microsoft/fluentui#25329] [General] [answered]
Hi, Could you help please, how can I add an icon in the lower right corner of the Persona image, instead of presence? For example, I want to add an icon.png of a medal there, as shown in the screenshot.
![Screenshot_541](https://user-images.githubusercontent.com/50998944/197193704-b948b0e9-12cc-459c-bd5d-7c864c859ef6.png)

`
<Persona
        {...examplePersona}
        size={PersonaSize.size72}
        presence={}
      />
`

**Comment by micahgodbolt** [comment, marked as answer]
there is no current way to override the rendering of the presence badge. You would need to render that custom badge on your own and then manually place it where you wanted it.
